src/Form/SiteType.php

```diff
@@ -19,6 +19,31 @@ class SiteType extends AbstractType
     public const DOWNLOAD_DISABLED = 'disabled';
     public const DV_HYBRID = 'hybrid';
 
+    public static $siteChoices = [
+        'status' => [
+            'Active'=> 1,
+            'Inactive' => 0
+        ],
+        'dv_module' => [
+            'Default (Based on HOS selection)' => null,
+            'DV Hybrid (Abbreviated PM Form + Kit)' => self::DV_HYBRID
+        ],
+        'centrifuge_type' => [
+            '-- Select centrifuge type --' => null,
```

Comment:
I like the way you are organizing this to make these mappings available outside the form. While you are doing this, could you remove this placeholder from the choices array and add it to the form field as a placeholder configuration?